src/components/BackButton.tsx

```diff
@@ -1,15 +1,8 @@
 import { Pressable, StyleSheet, Text } from "react-native";
+import React from 'react';
```

Comment:
to be able to remove this import statement, can i have you rebase to main? this can be done using `git rebase main`